src/vs/workbench/contrib/searchEditor/browser/searchEditor.contribution.ts

```diff
@@ -538,3 +540,36 @@ registerAction2(class OpenSearchEditorAction extends Action2 {
 	}
 });
 //#endregion
+
+//#region Search Editor Working Copy Editor Handler
+class SearchEditorWorkingCopyEditorHandler extends Disposable implements IWorkbenchContribution {
+
+	constructor(
+		@IInstantiationService private readonly instantiationService: IInstantiationService,
+		@IWorkingCopyEditorService private readonly workingCopyEditorService: IWorkingCopyEditorService,
+	) {
+		super();
+
+		this.installHandler();
+	}
+
+	private installHandler(): void {
+		this._register(this.workingCopyEditorService.registerHandler({
+			handles: async workingCopy => {
+				return workingCopy.resource.scheme === SearchEditorConstants.SearchEditorScheme;
+			},
+			isOpen: async (workingCopy, editor) => {
+				return editor instanceof SearchEditorInput && isEqual(workingCopy.resource, editor.modelUri);
```

Comment:
Again I assume that a search editor `modelUri` always matches the backup `search-editor:/...` URI.